float64 newX
float64 newY
float64 newTheta
---
